lib/jekyll-paginate-v2/generator/paginationModel.rb

```diff
@@ -251,17 +251,23 @@ def paginate(template, config, site_title, all_posts, all_tags, all_categories,
           paginated_page_url = config['permalink']
           first_index_page_url = ""
           if template.data['permalink']
-            first_index_page_url = template.data['permalink']
+            first_index_page_url = File.join(template.data['permalink'],"")
```

Comment:
File.join(something,"") appends a slash, in case it's missing. That should fix any potential double-slash problem.